src/components/PasswordField/PasswordField.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import styles from './PasswordField.module.scss';
+import hide from '../../assets/hidePasswordIcon.png';
+import show from '../../assets/showPasswordIcon.png'; 
+
+/**
+ * Parameters for the password field
+ */
+interface PasswordFieldProps {
+    /**
+     * String of the header for the textbox.
+     * @example
+     * header={"Enter Password"}
+     */
+    header: string;
+    /** Variable to set the input value of the password. */
+    setInputValue: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
this should be the same as the other one something like (value: string) => void. maybe you just missed it

Reply:
Thanks for catching that, I did miss it.